File: src/main/scala/util/impl/CashTransactionImpl.scala
package util.impl

import util.CashTransaction

class CashTransactionImpl extends CashTransaction {

  def findAll() = {
    println("find all cash transactions")
    "[]"
  }

  def find(id: String) = {
    println("find cash transaction " + id)
    "{}"
  }
}

File: src/main/scala/util/impl/CashTransactionImpl.java
package util.impl;

import util.CashTransaction;

public class CashTransactionImpl implements CashTransaction {
    @Override
    public String findAll() {
        System.out.println("find all cash transactions");
        return "[]";
    }

    @Override
    public String find(String id) {
        System.out.println("find cash transaction " + id);
        return "{}";
    }
}
